crates/net/network/src/session/mod.rs

```diff
@@ -77,7 +78,7 @@ pub struct SessionManager<N: NetworkPrimitives> {
     /// The secret key used for authenticating sessions.
     secret_key: SecretKey,
     /// The `Status` message to send to peers.
-    status: Status,
+    status: StatusMessage,
```

Comment:
same problem as with the NetworkConfig,

we can also introduce the two additional fields, for earliest,latest in additon,

maybe we should introduce a new super type for all status fields, which is a superset of status and status69?

Reply:
in latest commit https://github.com/paradigmxyz/reth/commit/8b1f040f3356cc766cba3a6f7463fd1b3923d7b0, I went with the unified status and status69 approach. The concept is that we store the superset of field in UnifiedStatus for use locally. and then, only use StatusMessage for rlp encoding and sending in eth_handshake. I went with this approach because it seems more extensible overtime. should be as easy as adding new fields to UnifiedStatus for future wire protocol version. additionally, nice not to have to track additional fields latest and earliest

@mattsse, lmk if like this approach or if prefer to keep things as legacy Status + two additional fields. if should proceed, I'll start on test refactoring and additionally see if I can uncover any edge cases